builder/builder-git.c

```diff
@@ -56,6 +56,19 @@ git_get_mirror_dir (const char     *url_or_path,
   g_autoptr(GFile) git_dir = NULL;
   g_autofree char *filename = NULL;
   g_autofree char *git_dir_path = NULL;
+  GPtrArray *sources_dirs = NULL;
+  int i;
+
+  sources_dirs = builder_context_get_sources_dirs (context);
+  for (i = 0; i < sources_dirs->len; i++)
+    {
+      GFile* sources_root = g_ptr_array_index (sources_dirs, i);
+      g_autoptr(GFile) git_dir = g_file_get_child (sources_root, "git");
```

Comment:
I did rename the inner loop variables for now, so we do not have shadow variables.